code/modules/organs/internal/kidneys.dm

```diff
@@ -1,10 +1,10 @@
 /obj/item/organ/internal/kidneys
```

Comment:
kidneys but only refers to left

Reply:
This is what we do with left/right arms, it's normal practice.